packages/common/src/requests/update_tag_name_request.ts

```diff
@@ -0,0 +1,4 @@
+import { InferType } from "yup";
+import { updateTagNameSchema } from "../validations/update_tag_name_schema";
+
+export type UpdateTagNameRequest = InferType<typeof updateTagNameSchema>;
```

Comment:
Maybe just remove the name part incase theres other stuff we can update on it in the future, no need to be specific

```suggestion
export type UpdateTagRequest = InferType<typeof updateTagSchema>;
```